### DISCUSSION TITLE
Ideas Wanted about Chrome's Side Panel Feature

### DISCUSSION THREAD

**Discussion by okaryo** [okaryo/TabTabTab#96] [Ideas]
Hi 👋,

As some of you might know, starting from version 114, Chrome has introduced a new feature called 'Side Panel'. This functionality can be extended by Chrome extensions, and we've been considering how we could leverage this within TabTabTab.

<img src="https://github.com/okaryo/TabTabTab/assets/44517313/6ebf9817-fd6b-428d-bc88-fc9fb8170935" width="320px" />

https://developer.chrome.com/docs/extensions/reference/sidePanel/

If you have any great ideas or use-cases for the Side Panel, we'd love to hear them! Your creative suggestions would be immensely valuable to us in determining how to best utilize this new feature.

To get the discussion started, here are some initial ideas I've brainstormed:

* Presets of multiple tab groups, which can be opened from the Side Panel
* Displaying statistical data related to tab usage
* Showing recently opened tabs
* Showing recently closed tabs
* Allowing customization of the display order of the above features from the settings page

Feel free to build upon these ideas, or suggest completely new ones. We look forward to hearing from you and can't wait to see what we can create together with this new Chrome capability.

**Comment by FreePDF64** [comment]
I hope that my Edge Browser will be supported as well ;)
